neurodocker/cli/minify/tests/test_minify.py

```diff
@@ -8,11 +9,22 @@
 docker = pytest.importorskip("docker", reason="docker-py not found")
 
 
-@pytest.mark.skip(reason="ptrace no longer supported under docker")
+def _arm_on_mac():
+    """Return True if on an ARM processor (M1/M2) in macos operating system."""
+    is_mac = platform.system().lower() == "darwin"
+    is_arm = platform.processor().lower() == "arm"
+    return is_mac and is_arm
```

Comment:
do you want to import this from the cli?

Reply:
yes that would make more sense. let me push that commit.